docs/manual/ru/introduction/Installation.html

```diff
@@ -143,22 +139,23 @@ <h2>Примеры</h2>
 	</code>
 
 	<p>
-		Если three.js был установлен с CDN, используйте тот же CDN для установки других компонентов:
+		Если three.js был установлен с CDN, используйте тот же код, но с `three/addons/` в карте импорта.
 	</p>
 
 	<code>
 		&lt;script type="module">
 
-		import { OrbitControls } from 'https://unpkg.com/three@&lt;version&gt;/examples/jsm/controls/OrbitControls.js';
+		import * as THREE from 'three';
```

Comment:
I think we also need an import map here.

Reply:
thanks for the comment. corrected it.